src/mangata-rococo.js

```diff
@@ -0,0 +1,161 @@
+import '@oak-network/api-augment';
+import _ from 'lodash';
+import moment from 'moment';
+import { cryptoWaitReady } from '@polkadot/util-crypto';
+import Keyring from '@polkadot/keyring';
+import confirm from '@inquirer/confirm';
+import TuringHelper from './common/turingHelper';
+import MangataHelper from './common/mangataHelper';
+import Account from './common/account';
+import { delay, listenEvents, readMnemonicFromFile } from './common/utils';
+import {
+    TuringStaging, MangataRococo,
+} from './config';
+
+/**
+ * README!
+ *
+ * 1. Unlike dev environment there’s no Sudo in Rococo, so we need to important a wallet first.
+ * a) Export a wallet from polkadot.js plugin, and save it as a file ./private/seed.json
+ * b) Run "PASS_PHRASE=<password_for_unlock> npm run mangata-rococo"
+ *
+ * 2. You need to have balances in both MGR and TUR on Mangata Rococo
+ * 3. You need to have balances in TUR on Turing Staging
+ */
+
+// Create a keyring instance
+const keyring = new Keyring({ type: 'sr25519' });
+
+/** * Main entrance of the program */
+async function main() {
+    await cryptoWaitReady();
+
+    console.log('Initializing APIs of both chains ...');
+    const turingHelper = new TuringHelper(TuringStaging);
+    await turingHelper.initialize();
+
+    const mangataHelper = new MangataHelper(MangataRococo);
+    await mangataHelper.initialize();
+
+    const turingChainName = turingHelper.config.key;
+    const mangataChainName = mangataHelper.config.key;
+    const turingNativeToken = _.first(turingHelper.config.assets);
+    const mangataNativeToken = _.first(mangataHelper.config.assets);
+
+    console.log(`\nTuring chain name: ${turingChainName}, native token: ${JSON.stringify(turingNativeToken)}`);
+    console.log(`Mangata chain name: ${mangataChainName}, native token: ${JSON.stringify(mangataNativeToken)}\n`);
+
+    console.log('1. Reading token and balance of account ...');
+
+    const json = await readMnemonicFromFile();
+    const keyPair = keyring.addFromJson(json);
+    keyPair.unlock(process.env.PASS_PHRASE);
```

Comment:
PASS_PHRASE=<password_for_unlock> npm run mangata-rococo

`PASS_PHRASE` is needed to unlock a wallet. This is the same behavior as polkadot.js wallet in Chrome.